### PR TITLE
`Reskin` 1789 Info Card Responsiveness update

### LINKED ISSUE #1789: Dashboard Info Card responsiveness Update
Noticed that when looking at this on smaller screens. (Smaller than 1300px) the three components escape parent container until the components wrap. 

Image

### PR DESCRIPTION
**Pull request by Da-Colon**
Fixes #1789 

- Move DAO InfoCard's Icons to top row
- Fixes Proposal Title textStyle
- Updates DAOInfoCard Header to be adjust to screen size easier

DAODashboard responsiveness

### CODE REVIEW

File: src/components/pages/DaoDashboard/Info/DaoInfoHeader.tsx [hunk 1]
```diff
@@ -107,35 +100,19 @@ export function DaoInfoHeader() {
...
                 <InfoGovernance />
               </InfoBox>
             </Box>
 
-            <Box
-              width={{ base: '100%', md: '33.3%', lg: '19%', xl: '20%' }}
-              ps={{ base: NONE, md: PAD }}
-              pe={{ base: NONE, md: PAD }}
-              pb={{ sm: PAD, md: NONE }}
-              pt={{ sm: PAD, lg: NONE }}
```
Comment (Da-Colon) [inline src/components/pages/DaoDashboard/Info/DaoInfoHeader.tsx:127]: I don't believe we need these anymore.

Comment (DarksightKellar) [reply src/components/pages/DaoDashboard/Info/DaoInfoHeader.tsx:127]: Boy am I glad to see these go.

**Review by DarksightKellar — APPROVED** [review]
Sweet!

**Comment by tomstuart123** [comment]
1) @Da-Colon responsiveness is feeling fire
2) Question, @xraystyle1980 , was the intention for the gear icon & favourites to now be above the DAOname for all screen sizes. Or just when it gets to mobile?

I may have missed a convo but just highlighting that the design file seems to point to them only going above for smaller screens
Screenshot 2024-05-17 at 15 16 54

Actually kinda like them above... wdyt

**Review by adamgall — APPROVED** [review]

**Pull request merged**